nixos/doc/manual/from_md/release-notes/rl-2205.section.xml

```diff
@@ -2387,6 +2387,15 @@
           <literal>tmux</literal>.
         </para>
       </listitem>
+      <listitem>
+        <para>
+          <literal>hardware.nvidia</literal> has a new option
+          <literal>open</literal> that can be used to opt in the
+          opensource version of NVIDIA kernel driver. Note that the
+          driver’s support for GeForce and Workstation GPUs is still
+          alpha quality.
```

Comment:
maybe we should also link to the official NIVIDA docs/announcement for more context